tests/topotests/lib/common_config.py

```diff
@@ -3973,3 +3975,30 @@ def verify_vrf_vni(tgen, input_dict):
 
     logger.debug("Exiting lib API: {}".format(sys._getframe().f_code.co_name))
     return False
+
+
+def required_linux_kernel_version(required_version):
+    """
+    This API is used to check linux version compatibility of the test suite.
+    If version mentioned in required_version is higher than the linux kernel
+    of the system, test suite will be skipped. This API returns true or errormsg.
+
+    Parameters
+    ----------
+    * `required_version` : Kernel version required for the suites to run.
+
+    Usage
+    -----
+    result = linux_kernel_version_lowerthan('4.15')
+
+    Returns
+    -------
+    errormsg(str) or True
+    """
+    system_kernel = platform.release()
+    if version_cmp(system_kernel, required_version) < 0:
+        error_msg = ('These tests will not run on kernel "{}", '
+            'they require kernel >= {})'.format(system_kernel,
+                required_version ))
+        return error_msg
+    return True
```

Comment:
This line is getting a flag because there's no newline?

Reply:
Yes.. If suggested, i will fix this. if not , pls review..